[master] Upgrade to latest dependencies

knative-automation opened the pull request:
Cron. -knative-prow-robot

/cc we don't use Prow
/assign we don't use Prow

Produced via:
```shell
./hack/update-deps.sh --upgrade --release v0.22
./hack/update-codegen.sh
```

Details:
```
bumping knative.dev/hack 8368e1f...f99a255:
  > f99a255 Schema tool for reflective schema generation. (# 51)
bumping knative.dev/pkg 8fbab7e...0f8d8de:
  > 0f8d8de Update common github actions (# 2051)
  > 185e333 Run downstream action on master branch too (# 2049)
  > 737401c upgrade to latest dependencies (# 2047)
  > 7f753ea Slash off 5s from TestAdmissionValidResponseForResource (# 2046)
  > 1002883 Add kn-plugin-admin to downstream tests (# 1983)
  > 04fdbd7 Add WaitForCacheSyncQuick and use it in tests (# 2045)
bumping knative.dev/eventing ef6d89e...37e7027:
  > 37e7027 Drop master references to knative repos (# 5036)
  > 32fc262 adding ping source smoke test (# 5033)
  > a4840f1 only build linux/amd64 (# 5032)
  > 65e14cf add UT for subscription delete/create failures (# 5031)
  > 8937481 break apart rekt tests from e2e tests (# 5025)
  > 149ebe6 Run kind actions on main branch as well (# 5024)
  > abd6f98 upgrade to latest dependencies (# 5022)
  > 6136d09 rename mtbroker dirs / package to just broker (# 5021)
  > e8bc8ed Change more internal usages of v1beta1 to v1 (# 5020)
  > 536e0aa sugar uses v1 versions of broker / trigger (# 5019)
  > f736471 Format markdown (# 5014)
  > 15c4916 upgrade to latest dependencies (# 5006)
  > 9d73c4d remove v1alpha.configmappropagations (# 5018)
  > 554c1cc Delete unused params in IMC controller reconciler (# 5015)
  > db81746 fix # 4994 use trigger's namespace only when subscriber's namespace is nil (# 4995)
  > dbf1859 Remove v1alpha1 channelable (# 5005)
```

n3wscott reviewed (dismissed):
/lgtm

Good job bot!

knative-automation committed:
upgrade to latest dependencies

bumping knative.dev/hack 8368e1f...f99a255:
  > f99a255 Schema tool for reflective schema generation. (# 51)
bumping knative.dev/pkg 8fbab7e...0f8d8de:
  > 0f8d8de Update common github actions (# 2051)
  > 185e333 Run downstream action on master branch too (# 2049)
  > 737401c upgrade to latest dependencies (# 2047)
  > 7f753ea Slash off 5s from TestAdmissionValidResponseForResource (# 2046)
  > 1002883 Add kn-plugin-admin to downstream tests (# 1983)
  > 04fdbd7 Add WaitForCacheSyncQuick and use it in tests (# 2045)
bumping knative.dev/eventing ef6d89e...37e7027:
  > 37e7027 Drop master references to knative repos (# 5036)
  > 32fc262 adding ping source smoke test (# 5033)
  > a4840f1 only build linux/amd64 (# 5032)
  > 65e14cf add UT for subscription delete/create failures (# 5031)
  > 8937481 break apart rekt tests from e2e tests (# 5025)
  > 149ebe6 Run kind actions on main branch as well (# 5024)
  > abd6f98 upgrade to latest dependencies (# 5022)
  > 6136d09 rename mtbroker dirs / package to just broker (# 5021)
  > e8bc8ed Change more internal usages of v1beta1 to v1 (# 5020)
  > 536e0aa sugar uses v1 versions of broker / trigger (# 5019)
  > f736471 Format markdown (# 5014)
  > 15c4916 upgrade to latest dependencies (# 5006)
  > 9d73c4d remove v1alpha.configmappropagations (# 5018)
  > 554c1cc Delete unused params in IMC controller reconciler (# 5015)
  > db81746 fix # 4994 use trigger's namespace only when subscriber's namespace is nil (# 4995)
  > dbf1859 Remove v1alpha1 channelable (# 5005)

Signed-off-by: Knative Automation <automation@knative.team>
